runtime/lua/nvim/health.lua

```diff
@@ -383,13 +383,27 @@ local function check_terminal()
   end
 end
 
+local function check_external_tools()
+  health.start('External Tools')
+
+  if vim.fn.executable('rg') == 1 then
+    local rg = vim.fn.exepath('rg')
+    local cmd = 'rg -V'
+    local out = vim.fn.system(vim.fn.split(cmd))
+    health.ok(('%s (%s)'):format(vim.trim(out), rg))
```

Comment:
Follow-up: the location of the used `rg` would be _really_ useful (`vim.fn.exepath()`) to debug `$PATH` issues.

Reply:
It's already being printed? it prints ripgrep v1.14.0 (/usr/bin/rg) on my system.

or are you saying the value of `grepprg` should use the exepath too?

Reply:
That's fine, then! (Sorry, should have tested before commenting.)